src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -129,6 +129,10 @@ public static int MainCore(string workingDirectory, string[] args)
                 // Parse the command
                 var command = parser.ParseCommandLine(args) ?? p.HelpCommand;
                 command.CurrentDirectory = workingDirectory;
+                if ((command is RestoreCommand restoreCommand && restoreCommand.NoCache == true) || (command is InstallCommand installCommand && installCommand.NoCache == true))
```

Comment:
Unless NoCache is a nullable bool, the "== true" is redundant.

```suggestion
                if ((command is RestoreCommand restoreCommand && restoreCommand.NoCache) || (command is InstallCommand installCommand && installCommand.NoCache))
```